sixtyfps_compiler/lib.rs

```diff
@@ -87,11 +93,22 @@ impl CompilerConfiguration {
             }
         };
 
+        let inline_all_elements = match std::env::var("SIXTYFPS_DISABLE_INLINING") {
+            Ok(var) => {
+                !var.parse::<bool>().unwrap_or_else(|_|{
+                    panic!("SIXTYFPS_DISABLE_INLINING has incorrect value. Must be either unset, 'true' or 'false'")
+                })
+            }
```

Comment:
```suggestion
        let inline_all_elements = match std::env::var("SIXTYFPS_INLINING") {
            Ok(var) => {
                var.parse::<bool>().unwrap_or_else(|_|{
                    panic!("SIXTYFPS_INLINING has incorrect value. Must be either unset, 'true' or 'false'")
                })
            }
```

I think it's good that this merge request defaults to inlining disabled for Rust/C++. In that case though I think the environment variable that we can use to debug the other case should not require a double-negation (I have to set DISABLE_INLINING to false in order to *enable* inlining).